app/databrowser/src/main/java/org/csstudio/trends/databrowser3/ui/Perspective.java

```diff
@@ -209,7 +209,7 @@ private void createContextMenu()
                     try {
                         action.call(plot.getPlot(), SelectionService.getInstance().getSelection());
                     } catch (Exception e1) {
-                        e1.printStackTrace();
+                        logger.log(Level.WARNING, "Failed to exectute " + action.getName() + " from databrowser plot.");
```

Comment:
Sorry, should still pass the exception so we get the stack trace:

```
                        logger.log(Level.WARNING, "Failed to exectute " + action.getName() + " from databrowser plot.", e1);
```
Note ", e1" at the end

Reply:
I thought I had included it... was using a search replace and got distracted with the CI failure

Reply:
sorry about this overlook